Dockerfile.dev

```diff
@@ -0,0 +1,12 @@
+# Develop using a known working environment.
+# @usage
+# run `make dev_container`
+FROM ocaml/opam:ubuntu-18.04-ocaml-4.13
+USER root
+RUN apt-get update && apt-get install -y libgmp-dev pkg-config
+USER opam
+RUN opam install --deps-only js_of_ocaml js_of_ocaml-lwt js_of_ocaml-compiler js_of_ocaml-ocamlbuild js_of_ocaml-toplevel js_of_ocaml-ppx js_of_ocaml-ppx_deriving_json js_of_ocaml-tyxml
+RUN opam install lwt reactiveData tyxml ppxlib cohttp-lwt-unix menhir ppx_expect yojson sexplib ocp-indent graphics
+RUN curl -fsSL https://fnm.vercel.app/install | bash
```

Comment:
Can we apt install node  instead ?

Reply:
done. i usually avoided apt before for node, because debian is still pegged to node 8 .x (8 major versions behind), but the node site now recommends it with a deb.nodesource.com as a deb-src :)